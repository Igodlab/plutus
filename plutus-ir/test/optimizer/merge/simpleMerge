(let
  (nonrec)
  (termbind
    (vardecl x (con integer)) (con 1)
  )
  (let
    (nonrec)
    (termbind
      (vardecl y (con integer)) (con 2)
    )
    y
  )
)
