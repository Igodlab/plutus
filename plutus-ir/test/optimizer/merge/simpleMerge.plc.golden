(let
  (nonrec)
  (termbind
    (vardecl unitval (all a (type) (fun a a))) (abs a (type) (lam x a x))
  )
  (termbind
    (vardecl unitval2 (all a (type) (fun a a))) (abs a (type) (lam x a x))
  )
  unitval2
)